### PR TITLE
Bailiff additions and sheriff changes

### PR DESCRIPTION
**Pull request by K4rlox**
## About The Pull Request

Changes Sheriffs description to clarify his role, and adds in two new Jobs; Bailiff and his assistants Councillors.

## Why It's Good For The Game
While k4rl may be submitting the PR for this, Headmin (Boardsofcanada) wrote the Descriptions, Composed the clothing, and requested for this to be done for me since I don't have any skills or experience in Git or scripting. This change in general is much more accurate to the setting, and breathes a new noble into the Manor's life cycle. This also cuts the sheriff off of some of his duties, while giving him new ones and giving a whole new role to assist the King and People in their activities.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/nobility/bailiff.dm [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/datum/job/roguetown/bailiff
+	title = "Bailiff"
+	flag = SHERIFF
```
Comment (ChungusGamer666) [inline code/modules/jobs/job_types/roguetown/nobility/bailiff.dm:3]: wrong flag

File: code/modules/jobs/job_types/roguetown/nobility/councillor.dm [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/datum/job/roguetown/councillor
+	title = "Councillor"
+	department_flag = NOBLEMEN
```
Comment (ChungusGamer666) [inline code/modules/jobs/job_types/roguetown/nobility/councillor.dm:3]: no job flag?

**Review by ChungusGamer666 — CHANGES_REQUESTED** [review]
These jobs feel awfully bloaty and pointless but before we discuss that, do the requested changes

File: code/modules/jobs/job_types/roguetown/nobility/bailiff.dm (hunk 1 above)
Comment (K4rlox) [reply code/modules/jobs/job_types/roguetown/nobility/bailiff.dm:3]: some other jobs including the hand use the shreiff flag

**Comment by K4rlox** [comment]
> These jobs feel awfully bloaty and pointless but before we discuss that, do the requested changes

DM boards_of_canada for that
despite i made the PR, its not really my PR
it was a direct addition request from the headmin, they were the ones who made the jobs

File: code/modules/jobs/job_types/roguetown/nobility/bailiff.dm (hunk 1 above)
Comment (K4rlox) [reply code/modules/jobs/job_types/roguetown/nobility/bailiff.dm:3]: image

File: code/modules/jobs/job_types/roguetown/nobility/councillor.dm (hunk 2 above)
Comment (K4rlox) [reply code/modules/jobs/job_types/roguetown/nobility/councillor.dm:4]: added, also fixed keyring

**Comment by OrbitalParadise** [comment]
usually the king handles whatever dispute there is, feels like a nerf to that if the guards just bring them to the court instead. Maybe make this into a courtroom for witch hunters

**Comment by K4rlox** [comment]
> usually the king handles whatever dispute there is, feels like a nerf to that if the guards just bring them to the court instead. Maybe make this into a courtroom for witch hunters

not the right person to tell this buddy

**Comment by YosemiteYam** [comment]
This is such a pointless PR and introduces more rolebloat and needlessly divides the crown faction. The sheriff is THE military head and enforces the law, the king SETS the law and if he wants to he can preside over judgement. We don't need a specific judge role with guards for that, and we don't need to nerf the sheriff's role either. We already have one role (dungeonkeeper) that is not used enough because people don't bring in prisoners, we don't need bailiff/counciller roles too. Also these descriptions are so bad, iterating that one is noble while the sheriff is not = unnecessary and creates discord. Seems to me that the judge role is for people who failed to roll king and just want to larp as King-Lite. 

The sheriff is the head of military (head of security) role and a minor noble, the king is the captain/lawgiver/judge role, we don't need to complicate this any more and make the crown more dysfunctional than it already is by giving a whole sub-faction of law-men when people barely get "judged" since there's no point to. The most common crimes are pickpocketing and assault and we have a whole garrison, sheriff, dungeonkeeper and a king to figure out anything more complex than that (never happens)

I hope the maintainers don't allow this

**Comment by K4rlox** [comment]
> This is such a pointless PR and introduces more rolebloat and needlessly divides the crown faction. The sheriff is THE military head and enforces the law, the king SETS the law and if he wants to he can preside over judgement. We don't need a specific judge role with his own (useless and role bloat) assistants for that, and we don't need to nerf the sheriff's role either. We already have one role (dungeonkeeper) that is not used enough because people don't bring in prisoners, we don't need bailiff/counciller roles too. Also these descriptions are so bad, iterating that one is noble while the sheriff is not = unnecessary and creates discord. Seems to me that the bailiff/judge role is for people who failed to roll king and just want to larp as King-Lite. If the king in a round wants to set specific councillors to preside over the maybe one criminal case that isn't some beggar twisting peoples dicks instead of doing it himself, he can appoint some or literally just use his hand, who most of the time doesn't have shit to do anyway.
> 
> The sheriff is the head of military (head of security) role and a minor noble, the king is the captain/lawgiver/judge role, we don't need to complicate this any more and make the crown more dysfunctional than it already is by giving a whole sub-faction of law-men when people barely get "judged" since there's no point to. The most common crimes are pickpocketing and assault and we have a whole garrison, sheriff, dungeonkeeper and a king to figure out anything more complex than that (never happens).
> 
> This is supposed to be a backwater kingdom where the law is direct and brutal, served by the King's goons or in rare cases the king himself, a law bureaucracy that will turn every beggar pickpocker case into a thirty minute court session is the silliest thing I can imagine for this server.
> 
> I hope the maintainers don't allow this

it was requested and written by the headmin

**Comment by ChungusGamer666** [comment]
Emoney likes this PR, I will give it a try

File: code/modules/jobs/job_types/roguetown/nobility/sheriff.dm [hunk 3]
```diff
@@ -11,7 +11,7 @@
 	"Aasimar")
 	allowed_sexes = list(MALE, FEMALE)
 	display_order = JDO_SHERIFF
-	tutorial = "Law and Order, your divine reason for existence. These animals are undeserving of your protection, for it is their sons and daughters roving the countryside with blade in hand; how many men have you lost this week just to the horrors in the woods alone? Are you the one to stand between this town and chaos, or will you fail it like they expect you to?"
+	tutorial = "You either inherited this role or were elected to it, you belong to no nobility and serve the king as loyal as you may, for he has paid you well. Enforce the law and ensure order prevails, you and the bailiff serve on the same authority; However, unlike the Bailiff you enforce the laws and maintain order in the town. You also oversee the general quality and conscription of Guards for the Town and Manor. The Bailiff can commandeer guards for his own protection, however he does not lead them as you do."
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/nobility/sheriff.dm:14]: Why is the Sheriff not a noble? Isn't that kind of weird?

File: code/modules/jobs/job_types/roguetown/nobility/bailiff.dm [hunk 4]
```diff
@@ -0,0 +1,56 @@
+/datum/job/roguetown/bailiff
+	title = "Bailiff"
+	flag = BAILIFF
+	department_flag = NOBLEMEN
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+
+	allowed_races = list("Humen",
+	"Humen",
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/nobility/bailiff.dm:10]: Humen, Humen, Aasimar. Might wanna fix this.

File: code/modules/jobs/job_types/roguetown/nobility/sheriff.dm (hunk 3 above)
Comment (K4rlox) [reply code/modules/jobs/job_types/roguetown/nobility/sheriff.dm:14]: ask the headmin

File: code/modules/jobs/job_types/roguetown/nobility/bailiff.dm (hunk 4 above)
Comment (K4rlox) [reply code/modules/jobs/job_types/roguetown/nobility/bailiff.dm]: fixed

**Comment by Real-MAGNUM** [comment]
Bailiff mapping is commented under bailiff key. Oops.

**Comment by Real-MAGNUM** [comment]
Do not merge this PR. I need(want) to do some stupid stuff because I dont want to fight versions of roguetown.dmm that werent converted with mapmerger2. I'll merge it when the conflicts are sorted, I'm ready.

**Pull request merged**